src/components/Input/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react"
+import PropTypes from "prop-types"
+import { InputCls } from "./styles"
+import { cx } from "emotion"
+
+Input.propTypes = {
+  className: PropTypes.string,
+  onChange: PropTypes.func,
+  label: PropTypes.string,
+  value: PropTypes.string,
+  disabled: PropTypes.bool,
+  errorMessage: PropTypes.string
+}
+
+function Input({ className, onChange, label, value, disabled, errorMessage }) {
+  const [inputValue, setInputValue] = useState("")
```

Comment:
No need to keep the value on the input in the state. The value is kept by the parent that hosts the input

Reply:
so the value will be passed as a property from the element that contains the component?